doc/source/ray-core/patterns/out-of-band-object-ref-serialization.rst

```diff
@@ -0,0 +1,26 @@
+.. _ray-out-of-band-object-ref-serialization:
+
+Anti-pattern: Serialize ray.ObjectRef out of band
```

Comment:
cross-reference it in https://docs.ray.io/en/latest/ray-core/objects/serialization.html#serializing-objectrefs